tests/integration/targets/ntnx_ndb_vlans/tasks/create_vlans.yml

```diff
@@ -0,0 +1,411 @@
+---
+- debug:
+    msg: Start testing ntnx_ndb_vlans
+
+- name: create Dhcp ndb vlan
```

Comment:
Please test spec created using check_mode as well. This will identify any regression in first step itself without any calls to server.